### DISCUSSION TITLE
GitHub Education- veryfication

### DISCUSSION THREAD

**Discussion by Rino32** [community/community#57461] [GitHub Education]
### Hi everyone,

Hi everyone, 

I got an issue even if i complete a git hub profile: [https://prnt.sc/uwQYK9qknlyB](url)

Can somebody help me? I will appreciate that. 

Regards Rino

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
